src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -878,6 +878,11 @@
       "datatype": "bool",
       "value": "(appTheme == 'cupertino')"
     },
+    "useUnoLottie": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "(unoLottie || appThemeEvaluator == 'fluent')"
```

Comment:
Will this handle cases when Uno.UI.Toolkit is used with Material? I think in such case Lottie also needs to be present for the extended splash

Reply:
updated this so it only brings it in with Toolkit & Fluent since it will already be brought in with Material/Cupertino